common/common.hpp

```diff
@@ -90,6 +90,16 @@ static inline bool MapContainsStringKey(
 	return map.find(str) != map.end();
 }
 
+template <typename KeyType, typename ValueType>
+static inline vector<KeyType> GetMapKeyVector(const unordered_map<KeyType, ValueType> &map) {
+	vector<KeyType> ret;
+	ret.reserve(map.size());
```

Comment:
Suggestion: Reserve the size in the initialiser?

Reply:
I actually tried that. I did `vector<KeyType> ret(map.size())`. But that doesn't reserve space, it actually creates a vector of that size with default-initialized items.